### ISSUE TITLE
Pagination Controls replaced by Blade template engine code

### ISSUE THREAD

**Issue by mipopa** [firefly-iii/firefly-iii#8677]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

On the accounts page, transactions page, and anywhere where a list has pagination, the usual pagination controls (such as page numbers, previous, next, first, last pages) are replaced by some PHP code preceded by the “@“ symbol which seems to the the error control operator.

see example below:

![IMG_6930](https://github.com/firefly-iii/firefly-iii/assets/55201147/25ae6fa7-5747-414e-b1ee-cf7714ac114f)

### Debug information

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>v6.1.7 / v2.0.12 / 22 (exp. 22)</td></tr><tr><td>PHP version</td><td>8.3.2-1+0~20240120.16+debian11~1.gbpb43448 / apache2handler / Linux</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>2097152 (2 MB)</td></tr><tr><td>Database drivers</td><td><strong>*sqlite*</strong>, </td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>Europe/Amsterdam + Europe/Luxembourg</td></tr><tr><td>App environment</td><td>local, debug: false</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>notice, stack / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td></td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>never (never)</td></tr><tr><td>Mailer</td><td>log</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td></td></tr><tr><td>Session start</td><td>2024-03-01 00:00:00</td></tr><tr><td>Session end</td><td>2024-03-31 23:59:59</td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>en_US</td></tr><tr><td>User locale</td><td>en_US</td></tr><tr><td>Locale(s) supported</td><td>en_US.utf8: :white_check_mark:<br>en_US.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) EdgiOS/122 Version/13.0.3 Safari/605.1.15</td></tr></tbody></table>

### Expected behaviour

Well, the pagination controls should be show 

### Steps to reproduce

To reproduce on the transactions page, just
1. Create a few transactions
2. Access the list of transactions

### Additional info

Details about my installation:

- self-hosted and managed server (no containerization)
- Apache + PHP8.3
- SQLite db
- All necessary PHP modules installed

**Comment by JC5** [comment]
What you're looking at is Blade template code, not error control operators.

But I don't use those templates in my code. Could you start by upgrading to the latest version of Firefly III?

**Comment by mipopa** [comment]
Hi there. Thanks for looking into this issue. I just completed the upgrade to version 6.1.10 using the instruction on this page https://docs.firefly-iii.org/how-to/firefly-iii/upgrade/self-managed/.
I still see the same blade template code, of whatever you say it's called.
Everything else works fine, except for page navigation.

**Comment by JC5** [comment]
Weird. I'm already out of ideas, I've never seen this before. I can replace the template, but I'm not sure it it'll help since it's not even rendering. But let's see if clearing some caches works:

```
php artisan cache:clear
php artisan config:clear
php artisan view:clear
```

**Comment by mipopa** [comment]
Thanks for the hints. I tried those commands to clear the caches, but no improvement. I’ll try to install firefly on a different machine (different distribution) to see if my PHP installation has issues. I’ll report back here soon

**Comment by JC5** [comment]
Let me know if Firefly III v6.1.11 fixes it, I've dropped the offending code.

**Comment by mipopa** [comment]
I’ve been having hard time installing Firefly III v6.1.11 and 6.1.12 in completely separated directories and apache virtual servers, and still didn’t figure out what’s wrong.
I’m following [these installation instructions](https://docs.firefly-iii.org/how-to/firefly-iii/installation/self-managed/) (using the zip file, not composer), combined with [these instructions](https://docs.firefly-iii.org/references/faq/install/#i-want-to-use-sqlite) to use sqlite as database.
Somehow, when running `php artisan migrate --seed` I’m asked confirm the creation of a database called “firefly”, although I _touch_  already the ./storage/database/database.sqlite.

The end-result is HTTP 500 and the log says clearly that no database is found. I don’t know what sqlite database file Firefly expects and where it should stay located. If I knew this, I could just try to move/rename the sqlite database created as “firely” and see how it works.

**Comment by JC5** [comment]
> I don’t know what sqlite database file Firefly expects and where it should stay located. 

Like you say, it expects `./storage/database/database.sqlite`.

Does it work when you upgrade your current installation to (now) 6.1.12?

**Comment by mipopa** [comment]
I attempted an upgrade from the version v6.1.7 (where I noticed the issue described in this ticket) to version 6.1.12, by following [these instructions](https://docs.firefly-iii.org/how-to/firefly-iii/upgrade/self-managed/). I paid good attention to skipping the `storage` directory when unzipping firefly, by using the command switch `-x "storage/*"`. Also made sure the permissions and ownership are set correctly.

According to the same upgrade instructions that I mentioned above, the below commands need to be run to complete the upgrade:
`php artisan migrate --seed
php artisan firefly-iii:decrypt-all
php artisan cache:clear
php artisan firefly-iii:upgrade-database
php artisan firefly-iii:laravel-passport-keys
php artisan cache:clear`
Unfortunately, all of them fail with the below error and the upgrade can't complete, causing HTTP 500.
![image](https://github.com/firefly-iii/firefly-iii/assets/55201147/1a2040ea-3cb9-4a76-b071-454dd6f8df2a)

I'm out of ideas...

**Comment by JC5** [comment]
Sorry, so am I. This usually happens when the composer lock file and the vendor files are out of sync. That shouldn't happen with a fresh or updated installation.

**Comment by mipopa** [comment]
> Sorry, so am I. This usually happens when the composer lock file and the vendor files are out of sync. That shouldn't happen with a fresh or updated installation.

Well, with a fresh install I still got the failure that I showed in [my previous post](https://github.com/firefly-iii/firefly-iii/issues/8677#issuecomment-2015833931) because the `composer.lock` comes from the zip file that the new installation instructions recommend to use.

What I did in my case to make version 6.1.12 work, was to remove the `composer.lock` file that comes from the zip and run the command `composer install --no-dev` which downloaded the latest versions of all the dependent vendor library files and created a fresh new `composer.lock` file.

Now, I'm able to run v6.1.12 and I'm happy to say that the pagination controls are also working fine.
Great job, @JC5 !
